.gitignore

```diff
@@ -0,0 +1,2 @@
+.idea
```

Comment:
It's Jetbrains project's configuration directory (you know - WebStorm, PHPStorm, PyCharm, IntelliJ Idea, etc). Lot's of people have this and push it by mistake. I thought it would be a good idea (pun not intended) to exclude it to be sure.